static/app/views/performance/newTraceDetails/traceTree.tsx

```diff
@@ -276,12 +277,12 @@ export class TraceTree {
       if ('start_timestamp' in value && value.start_timestamp < traceStart) {
         traceStart = value.start_timestamp;
       }
-      if (
-        'timestamp' in value &&
-        typeof value.timestamp === 'number' &&
-        value.timestamp > traceEnd
-      ) {
-        traceEnd = value.timestamp;
+      if ('timestamp' in value && typeof value.timestamp === 'number') {
+        if (isTraceError(value)) {
```

Comment:
Oh wow, this is very awkward. Can you leave a comment saying that error timestamps are considered as start_timestamps? A test for this wouldn't hurt either :)

Reply:
Yes I should